src/components/Card.js

```diff
@@ -1,48 +1,55 @@
 export default class Card {
-  constructor({ name, link }, cardSelector, handleImageClick) {
+  constructor(
+    { name, link, _id },
+    cardSelector,
+    handleImageClick,
+    handleDeleteClick
+  ) {
     this._name = name;
     this._link = link;
+    this._id = _id;
     this._cardSelector = cardSelector;
     this._handleImageClick = handleImageClick;
+    this._handleDeleteClick = handleDeleteClick;
 
-    this._cardElement = this._getTemplate();
-    this._cardImageEl = this._cardElement.querySelector(".card__image");
-    this._cardTitleEl = this._cardElement.querySelector(".card__title");
-    this._trashBtnEl = this._cardElement.querySelector(".card__delete-button");
-    this._likeBtnEl = this._cardElement.querySelector(".card__like-button");
+    this._element = this._getTemplate();
+    this._image = this._element.querySelector(".card__image");
+    this._title = this._element.querySelector(".card__title");
+    this._deleteButton = this._element.querySelector(".card__delete-button");
+    this._likeButton = this._element.querySelector(".card__like-button");
   }
 
```

Comment:
* When I reload the page, all **black** hearts become **transparent** again. Please, check `isLiked` inside the `Card` class